demo/client/app.js

```diff
@@ -4,7 +4,7 @@ import './global.css';
 import { render } from 'react-dom';
 import { createStore, compose } from 'redux';
 
-import { createStoreWithRouter } from '../../src';
+import { routerForBrowser } from '../../src';
```

Comment:
Is `routerForBrowser` in a separate module?  It would make sense to import something like `../../src/router-for-browser` instead, since this will avoid pulling in the code for `routerForExpress`.